index.md

```diff
@@ -15,5 +15,6 @@ This is a guide on solving common issues when using ReVanced YT and ReVanced man
 ## Other resources
 
 - [Get notified for ReVanced Updates](https://discord.com/invite/cHHX7PSPux)
-<!-- - [Guide by my friend Hokora Yinphine MPP#6969 on discord](https://hokorayinphinempp.github.io/obsidian-git-sync/Revanced/1%20Start%20Here%21/Revanced%20Start/) -->
 - [Dummies guide on reddit](https://www.reddit.com/r/revancedapp/comments/xlcny9/revanced_manager_guide_for_dummies/)
+- [Taku's auto CLI to patch on computer](https://github.com/taku-nm/auto-cli)
+<!-- - [Guide by my friend Hokora Yinphine MPP#6969 on discord](https://hokorayinphinempp.github.io/obsidian-git-sync/Revanced/1%20Start%20Here%21/Revanced%20Start/) -->
```

Comment:
Since it is outdated, it makes sense to add a note that it is outdated

Reply:
you mean hokora's guide? It was commented out.

Reply:
Oh lol. I didnt realize that's what that syntax meant